ng-dev/release/publish/constants.ts

```diff
@@ -14,3 +14,12 @@ export const changelogPath = 'CHANGELOG.md';
 
 /** Default interval in milliseconds to check whether a pull request has been merged. */
 export const waitForPullRequestInterval = 10000;
+
+/**
+ * Maximum number of characters a Github release entry can use for its body.
+ *
+ * Note: Github does not specify an official limit for this, but based on local testing,
+ * Github limits switch between 25000 and 125000 characters. We use the lowest limit we have
+ * seen so far, as otherwise the limit can potentially be wrong and result in errors.
+ */
+export const githubReleaseBodyLimit = 25000;
```

Comment:
I will reach out to Github and see if we can find out the official number because I remember personally putting in a request that was way more than 125000 characters successfully in the past during my testing.

Reply:
For some additional metrics, the tooling major releases seem to have a count of around 50,000.  But those have been entered manually.  Maybe a fallback to manual entry could be an option if the limit is problematic?

Reply:
sounds good. Should we proceed with the 25k for now though? you can see in the screenshot above that it mentions 25k. Today I was seeing 125k limit. I tried multiple scenarios

Reply:
@clydin Yeah, that seems possible. thx for additional info. I think the caretaker always would have the option to go back and edit the release manually.

Reply:
I tried a bunch of local testing and was unable to get any error message other than 125k characters.  Hopefully Github will get back to me on my request.

Reply:
yeah, same for me today. Although the screenshot shows that it was 25k a week ago (this showed up when I tested the release tool for the COMP repo)